.github/workflows/ci.yml

```
@@ -79,12 +78,13 @@ jobs:
           echo "LD_LIBRARY_PATH=$LD_LIBRARY_PATH" >> $GITHUB_ENV
 
       - name: Create lmi directories
-        run: >
+        run: |
           for d in /opt/lmi /opt/lmi/print /etc/opt/lmi /srv/cache_for_lmi; do
             sudo mkdir --parents $d;
             sudo --preserve-env chown $NORMAL_UID $d;
           done;
```

Minor, but due to the change of `>` (which was, of course, wrong, thanks for fixing this) to `|` above, we can now do
```suggestion
          done
```
here.